event/listener.php

```diff
@@ -82,6 +87,9 @@ public function init_collapsible_categories()
 	 */
 	public function show_collapsible_categories($event)
 	{
+		// Add collapsible-categories language file
+		$this->user->add_lang_ext('phpbb/collapsiblecategories', 'collapsiblecategories');
```

Comment:
This should not be here. This event takes place inside a loop and gets repeated multiple times.

Also, since our other event above is being removed in another PR, and because this should be available for other extensions to possibly use, this lang stuff should be handled using the core.user_setup model:
https://github.com/phpbb-extensions/boardrules/blob/master/event/listener.php#L80-L87